lib/id3c/labelmaker.py

```diff
@@ -302,6 +330,10 @@ class CollectionsChildcareLayout(LabelLayout):
     'collections-uw-observed': CollectionsUWObservedLayout,
     'collections-uw-home': CollectionsUWHomeLayout,
     'collections-childcare': CollectionsChildcareLayout,
+    'collections-school-testing': CollectionsSchoolTesting,
+    'collections-apple-respiratory': CollectionsAppleRespiratory,
+    'collections-adult-family-home-outbreak': CollectionsAdultFamilyHomeOutbreak,
+    'collections-workplace-outbreak': CollectionsWorkplaceOutbreak,
```

Comment:
By convention, would be nice to append Layout to these variables name, (e.g. `CollectionsSchoolTestingLayout`,  `CollectionsAppleRespiratoryLayout`)

Reply:
Yes, indeed. Fixed.